### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by MechCoder**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,19 @@ using `acq_optimizer='lbfgs'`.
 fine grained control over the iterations.
 * Parallelize L-BFGS minimization runs over the acquisition function.
 * Implement weighted hamming distance kernel for problems with only categorical dimensions.
+* New acquisition function `gp_hedge` that probabilistically one of `EI`, `PI`
```
Comment (glouppe) [inline CHANGELOG.md]: +chose?

**Comment by glouppe** [comment]
Tests are failing, for some unrelated reason... :(

File: CHANGELOG.md (hunk 1 above)
Comment (betatim) [inline CHANGELOG.md]: "...probabilistically chooses one of..."

**Comment by MechCoder** [comment]
fixed

**Comment by betatim** [comment]
A little unhappy because I don't understand why 4546af6 should fix the failure. I think making the test simpler is a good idea. Do we need a new issue to investigate if this is a failure because somewhere some random number generator isn't seeded properly?

**Comment by betatim** [comment]
Just ran the tests locally on acbbc761e5 (current master) and they passed. Points to this being some Heisenbug 😢

**Comment by MechCoder** [comment]
Sorry, I forgot to mention. My assumption was that the behaviour of `rng.normal` changed.

**Comment by betatim** [comment]
You were right: https://github.com/scikit-optimize/scikit-optimize/issues/318#issuecomment-279220970 :smiley:

**Pull request merged**